hermes-engine.podspec

```diff
@@ -24,13 +24,14 @@ Pod::Spec.new do |spec|
   # The podspec would be serialized to JSON and people will download prebuilt binaries instead of the source.
```

Comment:
I don't think React Native uses this file. It is pretty outdated, I think.

Reply:
It was needed to make the internal hermes test's work and properly build in Xcode